CHANGELOG_PENDING.md

```diff
@@ -146,6 +147,7 @@ Friendly reminder: We have a [bug bounty program](https://hackerone.com/tendermi
 - [state/privval] \#6578 No GetPubKey retry beyond the proposal/voting window (@JayT106)
 - [rpc] \#6615 Add TotalGasUsed to block_results response (@crypto-facs)
 - [cmd/tendermint/commands] \#6623 replace `$HOME/.some/test/dir` with `t.TempDir` (@tanyabouman)
+- [cleanup] \#6778 Remove redundant error plumbing
```

Comment:
I might be more of a rebel about this, but I sort of feel like changelogs shouldn't have things which don't impact interfaces or semantics just tend to hide things that users would care more about, and in that spirit, I'd omit this.

Reply:
I agree. I included it because it was listed in the guidelines, but I don't think it adds any real value. Reverted.